rd-net/Lifetimes/Lifetimes/OuterLifetime.cs

```diff
@@ -113,7 +114,7 @@ public static LifetimeDefinition Define(OuterLifetime lifetime, string? id = nul
 
         // Attach as nested to the parent lifetime
 //        if(!nested.IsTerminated) // Might have been terminated by FAtomic
-        lifetime.Def.Attach(nested); // Pass True: might be terminated async on another thread between our check and AttachNested body (example: Queue from another thread)
+        lifetime.Def.Attach(nested, true); // Pass True: might be terminated async on another thread between our check and AttachNested body (example: Queue from another thread)
```

Comment:
_(Minor)_ Any idea what this "Pass True" comment may be referring to?

After this change, it looks like it's talking about the second parameter of `Attach`, which, seemingly, cannot be?

Reply:
I think this comment refers to the commented ` if(!nested.IsTerminated)`. I think we can just delete these commented lines